src/shader-utils/assemble-shaders.js

```diff
@@ -101,9 +100,21 @@ function assembleShader(gl, opts = {}) {
 }
 
 export function assembleShaders(gl, opts) {
+  const vsSource = assembleShader(gl, opts);
```

Comment:
shaderCache is passed in as a property of `opts`. If this property does not exist, the function takes the original path and simply return shader source strings.